packages/webpack-cli/lib/webpack-cli.js

```diff
@@ -308,13 +308,24 @@ class WebpackCLI extends GroupHelper {
         const subject = allNames.filter((name) => {
             return args.includes(name);
         })[0];
+        const invalidArgs = args.filter(
+            (e) => !allNames.includes(e) && !e.includes('--color') && e !== 'version' && e !== '-v' && !e.includes('help'),
+        );
```

Comment:
We should encapsulate that logic into util/file

Reply:
new util file for invalid args =>  `utils/invalid-args`  ?

Reply:
`hasUnknownArgs()`